Dockerfile

```diff
@@ -22,12 +22,14 @@ RUN apk add --update --no-cache \
   uwsgi-python \
   uwsgi-logfile \
   musl \
-  musl-utils \
+  musl-utils>1.1.22-r3 \
   libtasn1 \
   \
   \
  && rm -rf /var/cache/apk/*
 
+RUN apk upgrade --no-cache
```

Comment:
Is that apk upgrade going to potentially upgrade other packages?

Could we target just the musl-utils with something like  `apk add --upgrade musl-utils`